makefile

```diff
@@ -31,11 +33,18 @@ run-sim2play: all
 	cd run; ./simulator --headless &
 	cd run; ./soccer -sim -y & ./soccer -sim -b
 
+debug: all
+	$(GDB) ./run/soccer
+
+debug-sim: all
+	-pkill -f './simulator --headless'
+	cd run; ./simulator --headless &
+	gdb ./run/soccer
```

Comment:
Sounds good.  Double-check that that works with `lldb` too - I'm not sure offhand if they have the same flags for that.